OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseUnitNavigationActivity.java

```diff
@@ -76,8 +76,17 @@ protected void onCreate(Bundle savedInstanceState) {
 
 
         pager.setOnPageChangeListener(new ViewPager.OnPageChangeListener() {
+
+            private Boolean  firstTime = true;
             @Override
             public void onPageScrolled(int position, float positionOffset, int positionOffsetPixels) {
+                if (firstTime) {
```

Comment:
Is this change making YouTube work or fixing another issue?

Reply:
edx-app uses a viewpager to change  to the next video or previous video with a gesture  and the viewpager only allows to define one video at the same time, so when the page is changed this generates an event where I destroy the previous video and create the new one, but  the first time (when the user go to the video from videos section) this event is not generated, so I have to create the video in a different way, and this change  is for that